### PR TITLE
Fix for sphinx autodoc

### PR DESCRIPTION
**Pull request by shashank-iitbhu**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

- Fix for sphinx autodoc not finding scribe_data module.
- modified `sys.path` in `conf.py`

### Related issue

- #57

### CODE REVIEW

**Comment by shashank-iitbhu** [comment]
Getting all the documentation for checkquery and utils.
Screenshot 2024-03-10 at 3 59 28 PM

File: docs/source/conf.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath(".."))
+sys.path.insert(0, os.path.abspath("../../src"))
```
Comment (wkyoshida) [inline docs/source/conf.py:21]: :raised_hands: Nice!!

Quick question - trying to catch up on our progress with the docs.. just wondering about the `extract_transform` and `load` stuff. Do those get addressed with this change as well or is the plan for us to tackle them later?

Comment (shashank-iitbhu) [reply docs/source/conf.py:21]: Can apply the same for `extract_transform` also, `extract_wiki.py` and `process_wiki.py` have some helper functions too.
not sure about `load`.

**Review by andrewtavis — APPROVED** [review]
Thanks for this, @shashank-iitbhu! It was working for me as well before this, but then looking at it this does seem to be the right path from conf.py :) Will test some things out with this merged in!

**Pull request merged**